### ISSUE TITLE
Fatal exception 0x80000003 

### ISSUE THREAD

**Issue by eaglenature** [EpicGames/raddebugger#131]
Debug build based on aa31afc. Launched application, hit F10 couple times then F11 and got exception:

![image](https://github.com/EpicGames/raddebugger/assets/2175244/6a8ed823-528d-46d5-a884-3e47e8718541)

![image](https://github.com/EpicGames/raddebugger/assets/2175244/3a7a24cc-723d-479c-9fdb-35d2cae6b0e3)

It looks like allocation size is huge. 

![image](https://github.com/EpicGames/raddebugger/assets/2175244/27c1a61d-a007-40f3-b0a1-ae47151bfa57)

![image](https://github.com/EpicGames/raddebugger/assets/2175244/85af4a71-2d76-4155-9e1f-454dc488e7a7)

![image](https://github.com/EpicGames/raddebugger/assets/2175244/6cacf162-3041-4358-b203-c73de97dec90)

![image](https://github.com/EpicGames/raddebugger/assets/2175244/debf25c1-99fb-4cca-8bde-556b0ae31416)

![image](https://github.com/EpicGames/raddebugger/assets/2175244/f2f51cf5-1264-4200-9d02-d0ba08d69d0b)

![image](https://github.com/EpicGames/raddebugger/assets/2175244/16533d11-8a57-4b2c-b234-eb3fc81b1bf6)

**Comment by eaglenature** [comment]
File df_gfx.c line:6677 looks suspicious. I've made a shot and replaced it with:
```
pted_eval.offset   = value_eval.offset;
MemoryCopyArray(pted_eval.imm_u128, value_eval.imm_u128);
```
and issue disappeared but its pretty blind shot though.

**Comment by ryanfleury** [comment]
This should be fixed by 509089d95d34ef0d562ca04039847b8da4371872 - let me know if you run into any more issues with it.
